src/epj.ProgressBar.Maui/ProgressBar.cs

```diff
@@ -159,6 +173,72 @@ private void DrawProgress()
             progressPaint.Color = ProgressColor.ToSKColor();
         }
 
+        if (UseSegment && SegmentCount > 0)
+        {
+            using SKPaint basePaint = GenerateSegment(progressPaint);
+        }
+        else
+        {
+            using SKPath progressPath = GenerateProgress(progressPaint);
+        }
+    }
+
+    private SKPath GenerateProgress(SKPaint progressPaint)
```

Comment:
The return value of this method is never used except for storing it in a temporary variable. This shouldn't be necessary, you could instead do the following:

```c#
using var progressPath = new SKPath();
```

That way, you don't need to return it, it's cleaned up after being drawn with `_canvas.DrawPath(progressPath, progressPaint);`